pennylane/devices/qubit/simulate.py

```diff
@@ -440,7 +440,8 @@ def simulate_tree_mcm(
 
     # `var` measurements cannot be aggregated on the fly as they require the global `expval`
     # variance_transform replaces `var` measurements with `expval` and `expval**2` measurements
-    circuit, variance_post_processing = variance_transform(circuit)
+    circuits, variance_post_processing = variance_transform(circuit)
+    circuit = circuits[0]
```

Comment:
Cool python syntax below:

```suggestion
    [circuit], variance_post_processing = variance_transform(circuit)
```

Reply:
Thanks!